apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -1506,10 +1506,17 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
                                     <Grid.Row columns={ 1 }>
                                         <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 10 }> 
                                             <Form.Field required={ true }>
-                                                <label className="pb-2">
+                                                <InputLabel
+                                                    htmlFor="tags-filled"
+                                                    disableAnimation
+                                                    shrink={ false }
+                                                    margin="dense"
+                                                    className="pb-2"
```

Comment:
Can we refrain from using these inbuilt padding classes ?

Reply:
Addressed with bbb71c09f74c657e0b9a95514b3d860b4f8532cb